### PR TITLE
document Tidal and vsphere password encryption

### PR DESCRIPTION
**Pull request by jmariomejiap**
## Changes

This PR adds Notes to communicate that we are encrypting Tidal Migrations and vSphere passwords.

### Getting Started with Tidal Tools

image

### Integrate Tidal Migrations with vSphere
image

### CODE REVIEW

File: pages/tidal tools/getstarted.md [hunk 1]
```diff
@@ -77,9 +77,13 @@ methods below.
 
 #### Configuration file
 
+
 Alternatively, you can use the `tidal config` command to manually set your
 credentials.
 
+It is  Configuration file is not encrypted.
```
Comment (prees1) [inline pages/tidal tools/getstarted.md:84]: grammars

File: pages/tidal tools/getstarted.md [hunk 2]
```diff
@@ -97,6 +101,7 @@ tidal:
   password: my_secure_password_here
   url: https://my_instance_name_here.tidalmg.com
 ```
+{% include note.html content="Your configuration file is not encrypted. Only your Tidal Migrations password and vSphere password are encrypted using AES 256 bit encryption." %}
```
Comment (prees1) [inline pages/tidal tools/getstarted.md:104]: consider starting with and emphasizing the positive.

```suggestion
{% include note.html content="Your Tidal Migrations password and vSphere password are encrypted, using AES 256 bit encryption when stored in the configuration file. The other parts of the configuration file are not encrypted." %}
```

File: pages/tidal tools/vsphere.md [hunk 3]
```diff
@@ -93,19 +93,24 @@ You can set your vSphere credentials with the following commands:
...
+  username: my_user_name_here  
   server: 1.1.1.1
 tidal:
-  email: my_user_name_here
-  password: my_secure_password_here
+  email: my_user_name_here  
   url: https://my_instance_name_here.tidalmg.com
 ```
+
+{% include note.html content="Your vSphere and Tidal password must be properly encrypted. In order to do so, you must use the tidal config set command."%}
```
Comment (prees1) [inline pages/tidal tools/vsphere.md:111]: consider saying it explicitly, can't set or edit passwords in the file.

```suggestion
{% include note.html content="You can not manually set or edit any passwords in the configuration file. They are stored here only as encrypted values. To persist a password, it must be added by using the `tidal config set` command."%}
```

**Review by prees1 — COMMENTED** [review]
good thinking on all the different parts in the guide that someone may be looking for this info.

File: pages/tidal tools/getstarted.md (hunk 1 above)
Comment (jmariomejiap) [reply pages/tidal tools/getstarted.md:84]: That line is no longer needed. Thanks for picking that up. 
```suggestion
```

**Comment by jmariomejiap** [comment]
> good thinking on all the different parts in the guide that someone may be looking for this info.

Thanks for the suggestions 💯

**Pull request merged**